removeDummies.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python
```

Comment:
This is the python script that will remove dummies. I would suggest that someone else checks it with a nifti image to double check that everything is as expected

Reply:
the voxel values are always "int" in the converted dicoms , right?

Reply:
ok finally took the time to check this. it works for me. 

Sent a PR to your branch to fix some style guide issues to abide by the pep8 standards